sw/device/lib/crypto/include/ecc.h

```diff
@@ -141,15 +141,21 @@ crypto_status_t otcrypto_ecdsa_keygen(const ecc_curve_t *elliptic_curve,
  * only for a custom curve. For named curves this field is ignored
  * and can be set to `NULL`.
  *
+ * The message digest must be exactly the right length for the curve in use
+ * (e.g. 256 bits for P-256), but may use any hash mode. The caller is
+ * responsible for ensuring that the security strength of the hash function is
+ * at least equal to the security strength of the curve. See FIPS 186-5 for
+ * details.
```

Comment:
Additionally, the user could also refer to the crypto lib user guide to get the security strength for hash/xof and ECC.  
[Security Strength](https://opentitan.org/book/doc/security/cryptolib/cryptolib_api.html#security-strength)
Not sure if it is worth adding to the header.